src/main/java/htsjdk/variant/vcf/VCFEncoder.java

```diff
@@ -201,33 +198,66 @@ else if ( val instanceof List ) {
 		return result;
 	}
 
+	private static double round(final double value, final int digits) {
+		final double x = Math.pow(10, digits);
+
+		return ((int)(value * x)) / x;
```

Comment:
this should probbly use Math.round rather than (int) so that we do not always round down.